src/libstd/collections/hash/set.rs

```diff
@@ -454,6 +454,25 @@ impl<T, S> HashSet<T, S>
         self.map.contains_key(value)
     }
 
+    /// Gets a value from a set.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #![feature(collection_member)]
+    /// use std::collections::HashSet;
+    ///
+    /// let mut set = HashSet::new();
+    ///
+    /// assert_eq!(set.insert(2), true);
+    /// assert_eq!(set.get_member(&2), Some(&2));
+    /// ```
+    #[unstable(feature = "collection_member",
+            reason="member stuff is unclear")]
+    pub fn get_member(&mut self, value: &T) -> Option<&T> {
```

Comment:
For sets I'd argue this can just be `get`, but I understand symmetry is a concern.